README.md

```diff
@@ -47,6 +47,10 @@ ssh localhost -p 3232
 
 ## Setup Instructions
 
+> **NOTE:** reverse_ssh requires Go **1.17.1** or higher. Please check you have at least this version via `go version`
```

Comment:
this is because i didn't have this version on my windows, kali and ubuntu instances, and without it you get compile errors about not knowing where embed.go is etc.